### ISSUE TITLE
Hide Menu Bar Icon on OSX

### ISSUE THREAD

**Issue by MartSerra** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1574]
**OS:**
OS X 10.11.6 El Capitan

**Issue Descriptions:**
Is it possible to hide the icon en Menu Bar? But not closing the application.

**Steps to Reproduce:**
An option for hide menu bar icon from _GooglePlayMusicDesktopPlayer_ without closing the application.

**Comment by MarshallOfSound** [comment]
When you say Menu Bar.  Can you clarify exactly what you mean?

Do you mean the Dock or the System Tray?

**Comment by MartSerra** [comment]
O my apologize, I mean hide the icon on system tray.
![screen shot 2016-09-03 at 12 03 13 pm](https://cloud.githubusercontent.com/assets/7231751/18226404/8745529a-71ce-11e6-9079-4c9554bcfc58.png)
And I would be awesome if the controls that are implemented on this menu tray icon could be placed on application dock.
![screen shot 2016-09-03 at 12 09 19 pm](https://cloud.githubusercontent.com/assets/7231751/18226435/6d6d5d1c-71cf-11e6-8a3d-53da722c7cb8.png)
 For example I attached a Spotify controls.
![screen shot 2016-09-03 at 12 05 30 pm](https://cloud.githubusercontent.com/assets/7231751/18226417/d62a6404-71ce-11e6-8713-4f6eab90a52c.png)
Thanks for your support @MarshallOfSound

**Comment by MarshallOfSound** [comment]
@MartSerra I've had the "please let me hide the system tray icon" discussion more times than I can count on the issues section on GitHub.

TLDR: We can't and we won't 😢
